samples/cloudwatch/README.md

```diff
@@ -1,73 +1,88 @@
-* Check the box next to `CloudWatchAgentServerPolicy`
-* Choose `Attach Policy`
-* Repeat for the role with `AWS service: ec2`
+Follow these steps to set permissions:
 
-If the `CloudWatchAgentServerPolicy` does not exist, see:
+1. Sign in to the AWS Management Console and open the IAM console.
+2. In the navigation pane of the console, select `Roles`.
+3. In the Search field, enter the name of the Amazon EKS cluster on which SAS Viya is deployed (such as `my-cluster`) to filter the roles listed.
+4. Select the role that contains `AWS service: eks` in the `Trusted entities` column.
+5. Select `Attach Policies`.
+6. Search for `CloudWatchAgentServerPolicy`. If the `CloudWatchAgentServerPolicy` does not exist, see:
 [Creating IAM Roles for Use with the CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/create-iam-roles-for-cloudwatch-agent-commandline.html).
+7. Select the check box next to `CloudWatchAgentServerPolicy`.
+8. Select `Attach Policy`.
+9. Repeat steps 4 through 8 for the role that contains `AWS service: ec2` in the `Trusted entities` column.  
 
 ### Identify Cluster and Region
 
-Run the following snippit to identify the cluster name and region. Replace the
+You must identify the Amazon EKS cluster on which SAS Viya is deployed and the region for the cluster. These values are needed to construct the correct endpoint for CloudWatch.
+
+Run the following code from the command line. Replace the
 values of `ClusterName` and `RegionName` with the values for your EKS cluster.
 
 ```bash
 ClusterName=my-cluster
 RegionName=us-east-1
 ```
 
-### Deploy CloudWatch Prometheus Agent
+### Deploy CloudWatch Agent for Prometheus
+
+Run the following code from the command line to deploy the CloudWatch agent for Prometheus metrics:
 
 ```bash
 curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/latest/k8s-deployment-manifest-templates/deployment-mode/service/cwagent-prometheus/prometheus-k8s.yaml | 
 sed "s/{{cluster_name}}/'${ClusterName}'/;s/{{region_name}}/'${RegionName}'/" | 
 kubectl apply -f -
 ```
 
-### Apply SAS Configuration
+### Apply SAS Viya Customizations
+
+This sample includes configuration files that transform the scraped SAS Viya Prometheus metrics into performance log events and then map the performance log events to Cloudwatch metrics.
+
+##### Configure Scraped Metrics
+
+Run this command to configure the metrics scraped by Prometheus and convert them into performance log events:
 
 ```bash
-# Prometheus scrape config
-# Scraped metrics will become performance log events
 kubectl apply -f sas-prom-config.yaml
-# Map performance log events to CloudWatch metrics
+```
+#### Map Performance Log Events to CloudWatch Metrics
+
+Run these commands to map the performance log events that were created from Prometheus metrics to CloudWatch metrics and restart the agent in order to apply the configuration changes:
+
+```bash
 kubectl apply -f sas-prom-cwagentconfig.yaml
```

Comment:
Remove this line to eliminate the merge conflict. We don't need to apply the yaml as is. The next line applies the same yaml, but 'fixed' with the correct replacement values.

Reply:
Line removed